src/configuration/validators/neovimValidator.ts

```diff
@@ -8,12 +8,31 @@ export class NeovimValidator implements IConfigurationValidator {
     const result = new ValidatorResults();
 
     if (config.enableNeovim) {
+      let triedToParsePath = false;
       try {
+        // Try to find nvim in path if it is not defined
+        if (config.neovimPath === '') {
+          const pathVar = process.env.PATH;
+          if (pathVar) {
+            pathVar.split(';').forEach(element => {
+              if (element.toLocaleLowerCase().includes('neovim')) {
+                const path = require('path');
+                config.neovimPath = element + path.sep + 'nvim';
+                triedToParsePath = true;
+                return;
+              }
+            });
+          }
+        }
         await promisify(execFile)(config.neovimPath, ['--version']);
       } catch (e) {
+        let errorMessage = `Invalid neovimPath. ${e.message}.`;
+        if (triedToParsePath) {
+          errorMessage += 'Tried to parse PATH.';
```

Comment:
nit: should we output the path we tried?  `config.neovimPath

Reply:
Sure, would be easy